switchman-inst-jobs.gemspec

```
@@ -26,7 +26,7 @@ Gem::Specification.new do |s|
   s.add_dependency "inst-jobs", ">= 2.4.9", "< 4.0"
   s.add_dependency "parallel", ">= 1.19"
   s.add_dependency "railties", ">= 6.1", "< 7.2"
-  s.add_dependency "switchman", "~> 3.1"
+  s.add_dependency "switchman", "> 3.1", "< 5.0"
```

Should this be `>=`?
```suggestion
  s.add_dependency "switchman", ">= 3.1", "< 5.0"
```